src/plugin/advancedCompare/index.js

```diff
@@ -0,0 +1,40 @@
+export default (option, dayjsClass) => {
+  const proto = dayjsClass.prototype
+  const oldSame = proto.isSame
+  const oldIsAfter = proto.isAfter
+  const oldIsBefore = proto.isBefore
+
+  const v = (l, r) => l.isValid() && r.isValid()
+
+  dayjsClass.prototype.isSame = function (o, u) {
+    if (!v(this, o)) return false
+
+    return (u && u === 'millisecond')
+      ? oldSame.bind(this)(o)
+      : this.startOf(u).valueOf() <= o && o <= this.endOf(u).valueOf()
+  }
+
+  dayjsClass.prototype.isAfter = function (o, u) {
```

Comment:
use `this.$utils().prettyUnit(u)`  to deal with 'ms', 'millisecond' and 'milliseconds'

Reply:
Right, my bad. Fixed with bba77e4e246effc0070fb5fb6ae4f8b8f8e30f96.